lib/runCall.js

```diff
@@ -417,7 +417,9 @@ module.exports = function (opts, cb) {
       callCtx.setRecipientaddr(toAddress)
     }
     callCtx.setInputdata(txData)
-    callCtx.setCallvalue(new Uint8Array(txValue))
+    var value = Buffer.from(txValue)
+    value = ensurePositive(value)
+    callCtx.setCallvalue(new Uint8Array(value))
```

Comment:
Let's either (1) rename `ensurePositive` to reflect that it changes the type or (2) change it so it keeps the type the same.

Then the double cast to `Uint8Array` will be avoided.

Reply:
pushed a change